Assets/Scripts/Structres/Projectile.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+using FishNet.Object;
+
+public class Projectile : NetworkBehaviour
+{
+    [SerializeField] private float _speed = 1f;
+    [SerializeField] private float _damage;
+
+    private Transform _target;
+
+    public Transform Target { set => _target = value; }
+
+    private void Awake() => Destroy(gameObject, 10f);
```

Comment:
I would make the timer before it gets destroyed a tweakable variable and I would also add a max distance variable which can be tweaked so we can easily balance the max distance we want towers to be able to shoot.